src/app/app.component.ts

```diff
@@ -730,9 +734,101 @@ export class AppComponent implements OnInit {
         }
         this.comics[comicIndex].styles.background = collectionColor.backgroundColor;
         this.comics[comicIndex].styles.color = collectionColor.textColor;
+
+        //  while we are here store the comics in reading order
+        this.comicsInReadingOrder.push(_.cloneDeep(this.comics[comicIndex]));
       });
     });
 
+    //  reposition the reading order comics horizontally
+    let horizontalReadingOrderPosition = 0;
+    _.each(this.comicsInReadingOrder, (comic) => {
+      horizontalReadingOrderPosition += VISUAL_BLOCK_SIZE;
+      comic.containerStyles['left.px'] = horizontalReadingOrderPosition;
+
+      /*
+       * Look up the volume and series for this comic and
+       * throw errors for obvious problems before proceeding.
+       */
+      const currentSeriesVolume = this.seriesVolumes[_.findKey(this.seriesVolumes, { id: comic.seriesVolumeId })];
+      if (!currentSeriesVolume) {
+        throw new Error(comic.seriesVolumeId + ' not found');
+      }
+      // this is a copy of the other positioning
+      /*
+       * Vertical positioning ensures that each seriesVolume gets
+       * its own row on the page. The exception is if a seriesVolume
+       * has not had any new issues for a whole page width, then we
+       * allow the next seriesVolume that is looking for a row, to slot in.
+       *
+       * This is a two-step process.
+       * First, in this block, we either use the current position for the
+       * seriesVolume or we use the globalVerticalPositionCounter to go onto
+       * the last row.
+       * Step two documented below.
+       */
+
+       // TO-DO: Paulene really needs to understand what's happening here
+       // so I added this line because I couldn't get things to line up.  Unsure what it should be reset to.
+      this.globalVerticalPositionCounter = 1;
+      // this if statement checks that the vertical position attribute of the currentSeriesVolume is not undefined
+      // I'm wondering when it would be undefined.
```

Comment:
It's `undefined` when it hasn't been defined yet. This is the part of the code that defines it, so that just means this loop hasn't happened for this series yet.